deployment/nether-deploy-quickstart.json

```diff
@@ -1,93 +1,256 @@
 {
-    /*
```

Comment:
The comments in this template should be left as is unless documented or explained why they need to be removed.